src/99-appendices/07-meg-systems.md

```diff
@@ -1,17 +1,17 @@
 # Appendix VII: preferred names of MEG systems
 
-Restricted keywords for Manufacturer field in the `*meg.json` file:
+Restricted keywords for Manufacturer field in the `*_meg.json` file:
 
--   `CTF`
--   [`Elekta/Neuromag`](https://docs.google.com/document/d/1FWex_kSPWVh_f4rKgd5rxJmxlboAPtQlmBc1gyZlRZM/edit#heading=h.a7ggx48p7aaf)
--   [`4D/BTi`](https://docs.google.com/document/d/1FWex_kSPWVh_f4rKgd5rxJmxlboAPtQlmBc1gyZlRZM/edit#heading=h.gy0kbzisg1f1)
--   [`KIT/Yokogawa/Ricoh`](https://docs.google.com/document/d/1FWex_kSPWVh_f4rKgd5rxJmxlboAPtQlmBc1gyZlRZM/edit#heading=h.2gmmxawyna7r)
--   `KRISS`
--   [`ITAB`](https://docs.google.com/document/d/1FWex_kSPWVh_f4rKgd5rxJmxlboAPtQlmBc1gyZlRZM/edit#heading=h.58whib3oq56y)
--   `Aalto/MEG–MRI`
+-   [`CTF`](06-meg-file-formats.md#ctf)
+-   [`Elekta/Neuromag`](06-meg-file-formats.md#neuromagelektamegin)
```

Comment:
I would include Megin here: [`Elekta/Neuromag/Megin`]

Reply:
But then keep the order historically consistent, i.e. `Neuromag/Elekta/Megin`, or the other way around. For FieldTrip I decided to converge on old/newer/current.

See also http://www.fieldtriptoolbox.org/getting_started/ctf/, http://www.fieldtriptoolbox.org/getting_started/bti/, http://www.fieldtriptoolbox.org/getting_started/neuromag/, to which I just added a short "historical" section at the top.

Reply:
Cool, I really like these historical sections ... they are important for new people to understand what all of these names are about :-)

Now that we are fixing names, what about `KIT/Yokogawa/Ricoh`? ... in the FT wiki, there are separate pages for:

- ricoh: http://www.fieldtriptoolbox.org/getting_started/ricoh/
- yokogawa: http://www.fieldtriptoolbox.org/getting_started/yokogawa/

yet they are combined here in the BIDS spec. Also: I cannot find an entry for KIT in the FT wiki

@robertoostenveld

Reply:
ping @robertoostenveld, in case you missed my question

Reply:
@KiwamuK, the Ricoh engineer that currently maintains the support for their system in FieldTrip, preferred to keep the two website pages separate. Afaik the file formats are still the same, hence I propose to specify it as `KIT/Yokogawa/Ricoh`, in line with the others. I do not have information about the MEG system prior to Yokogawa being involved.